quickwit/quickwit-config/src/quickwit_config/serialize.rs

```diff
@@ -486,7 +486,7 @@ mod tests {
         .await
         .unwrap_err();
         assert!(format!("{parsing_error:?}")
-            .contains("unknown field `max_num_concurrent_split_searchs`"));
+            .contains("unknown field `max_num_concurrent_split_searches_with_typo`"));
```

Comment:
I don't know how this test managed to be successful

Reply:
The sample is also renamed: https://github.com/quickwit-oss/quickwit/pull/2927/files/6181a93cbd1006b6fe457f10ae789a1f4c48bf4d#diff-cb0bc5c6a7c42d89c3a510f4bb52dbfef2d15f42c3fc478c27aca6b720e560ba

Reply:
Ah wait damn `searchs` is still spelt wrong